packages/dx-react-grid-demos/src/demo-sources/grid-fixed-columns/with-selection.jsxt

```diff
@@ -65,9 +63,8 @@ export default class Demo extends React.PureComponent {
           <TableHeaderRow />
           <TableSelection />
           <TableFixedColumns
-            leftColumnTypes={leftColumnTypes}
-            leftColumnNames={leftColumnNames}
-            rightColumnNames={rightColumnNames}
+            leftColumns={leftColumns}
+            rightColumns={rightColumns}
```

Comment:
The sentence `You can also use the leftColumnTypes and rightColumnTypes properties to fix service columns, such as Selection Column or Editing Column.` is keeping in the doc

Reply:
Fixed